charts/docker-mailserver/templates/service.yaml

```
@@ -137,6 +137,9 @@ spec:
     {{- end }}
 
   type: {{ default "ClusterIP" .Values.service.type }}
+  {{- if and .Values.service.privateIp (has (default "ClusterIP" .Values.service.type) (list "ClusterIP" "NodePort")) }}
```

Might be clearer to create a variable for the service type, e.g.
```suggestion
  {{- $serviceType := default "ClusterIP" .Values.service.type -}}
  {{- if and .Values.service.privateIp (or (eq $serviceType "ClusterIP") (eq $serviceType "NodePort")) }}
```

---

This is no longer relevant with the latest changes.